web/src/auth/LoginPage.js

```diff
@@ -156,8 +156,8 @@ class LoginPage extends React.Component {
       values["type"] = "saml";
     }
 
-    if (this.state.owner !== null && this.state.owner !== undefined) {
-      values["organization"] = this.state.owner;
+    if (this.state.organization !== null && this.state.organization !== undefined) {
```

Comment:
@anchaogit has `this.state.organization` been defined anywhere?

Reply:
![img_c5c187f97089d4c67224a1d61fa7eb83](https://user-images.githubusercontent.com/8927881/196580984-50ac262d-3aca-4628-aa99-f4f13e68e2a5.png)

Yesterday I submitted the wrong code, and today I resubmitted it. The location of variable definition is shown in the figure above. I am a back-end developer, and I saw it on the web console